db/seeds/playbook.seeds.rb

```diff
@@ -113,7 +120,12 @@
   explained, even now. Perhaps only one event in a thousand is
   true weirdness — but I’ll investigate them all to find it.'
  }].each do |playbook|
-  Playbook.find_or_initialize_by(name: playbook[:name]).update!(playbook)
+  pb = Playbook.find_or_initialize_by(name: playbook[:name])
+  if playbook.key?(:categories)
+    categories = playbook.delete(:categories)
+    pb.category_list = categories
+  end
+  pb.update(playbook)

```

Comment:
See comment above.  We needed a way for the playbooks to know which categories they had so that if a gear could be in multiple categories it wouldn't just pick whatever category was first.  I built out tags to indicate them and saved them in the category_list for the playbook.